### ISSUE TITLE
API (POST) call example from a Unity 3D game

### ISSUE THREAD

**Issue by azureus89** [224-1/unity-api-call-example#1]
Build a simple example that demonstrates API calls being made to a server (url will be provided to you) from a unity3D game.

- Create a new unity3d game project with a single object that sends an makes an API request when it is clicked
- Demonstrate a few more events - keypress etc
- Compile unity3d game to webGL and demonstrate call success from a webgl game in a static HTML page
- Demonstrate call over websocket

Please clarify any doubts before starting.

**Comment by azureus89** [comment]
@sujeetboran use the same issue and keep adding comments here because it is all related to this issue.
Dont have to create new issue for everything because it is all part of the this issue.
